Tests/StoreKitUnitTests/TestHelpers/AvailabilityChecks.swift

```diff
@@ -74,4 +74,10 @@ enum AvailabilityChecks {
         #endif
     }
 
+    static func skipIfXcode14orEarlier() throws {
+        #if swift(<5.9)
+        throw XCTSkip("Test for for Xcode 14 or earlier")
```

Comment:
This should be 
```suggestion
        throw XCTSkip("Test for for Xcode 15 or earlier")
```

Reply:
Renoved this helper as it's no longer needed